tests/framework/db/mssql/SchemaTest.php

```
@@ -208,4 +210,171 @@ public function testGetPrimaryKey(): void
+
+    public function testNullDefaultValueColumn(): void
+    {
+        $schema = $this->getConnection(true, true)->getSchema();
+        $table = $schema->getTableSchema('null_values');
+        $this->assertNull($table->getColumn('var3')->defaultValue);
+        $this->assertNull($table->getColumn('stringcol')->defaultValue);
+    }
+
+    public function testInsertWithCompositePrimaryKey(): void
+    {
+        $db = $this->getConnection(true, true);
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)